invenio_rdm_migrator/state/base.py

```diff
@@ -55,11 +65,6 @@ def all(self):
         """Get all the data from the state."""
         return self._data.values()
 
-    @abstractmethod
-    def _validate(self, data):
```

Comment:
When using JSON and in-memory dictionaries the structure (k,v pairs) had to be validated. However, this will now be done upon insertion on the sqlite db. Therefore, there is no need to validate neither on add neither on load.